packages/react-storage/src/components/StorageBrowser/composables/DataTable/dataCells/ButtonDataCell.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+
+import {
+  ButtonElement,
+  ButtonElementVariant,
+  IconElement,
+  IconVariant,
+} from '../../../context/elements';
+import { CLASS_BASE } from '../../../views/constants';
+
+export interface ButtonDataCellProps {
+  content: {
```

Comment:
Sure there it probably a good reason for nesting the presentational values in `content` but does feel a bit extranous

Reply:
`content` is a sibling of key and type and so all three properties are consistent across data cells and gives us a consistent pattern and place to target data